birdhouse/docker-compose.yml

```diff
@@ -213,9 +233,8 @@ services:
       - "48983:9001"
     volumes:
       - /data/solr:/data
-    volumes_from:
-      - mongodb
```

Comment:
From what I understood from our meeting, the removal of `volumes_from` directive was because of the update to format version `3.4`, not because we want it or Weaver needs it?

Reply:
Agreed, this looks harmless but should be tested to be sure.  Has you or Mat gone over the failed build to determine the error is not related to this?  Currently only the ESGF notebook should be intermittently failing because their server seems flaky but everything else should be green.